comp/core/status/component.go

```diff
@@ -21,8 +21,8 @@ const CollectorSection string = "collector"
 type Component interface {
 	// Returns all the agent status information for the format type
 	GetStatus(format string, verbose bool, excludeSection ...string) ([]byte, error)
-	// Returns only the agent status for the especify section and format type
-	GetStatusBySection(section string, format string, verbose bool) ([]byte, error)
+	// Returns only the agent status for the specified sections and format type
+	GetStatusBySection(sections []string, format string, verbose bool) ([]byte, error)
```

Comment:
Should we rename the function to `GetStatusBySections` ?

Reply:
Sure, that is easy

Reply:
see it in last commit